test/specs/orgBrowser.e2e.ts

```diff
@@ -113,60 +110,58 @@ describe('Org Browser', async () => {
   step('Refresh Org Browser and check MyClass is there', async () => {
     utilities.log(`${testSetup.testSuiteSuffixName} - Refresh Apex Classes`);
     // Check MyClass is present under Apex Classes section
-    await utilities.executeQuickPick('View: Show Org Browser', utilities.Duration.seconds(5));
+    await utilities.openOrgBrowser(utilities.Duration.seconds(1));
```

Comment:
Default is already 1 second.

```suggestion
    await utilities.openOrgBrowser();
```